mixes/containers/mix.js

```diff
@@ -17,7 +17,7 @@ class MixContainer extends React.Component {
     if (!mix) { return null }
 
     console.log('mix', mix)
-    const title = isLoading
+    const $title = isLoading
```

Comment:
idk, i don't name them any differently. isn't this just a boolean? if it's an element, i often name them like `loadingElement`.